gdk4/sys/src/lib.rs

```diff
@@ -4206,6 +4209,7 @@ extern "C" {
     //=========================================================================
     // GdkToplevelSize
     //=========================================================================
+    pub fn gdk_toplevel_size_get_type() -> GType;
```

Comment:
This lacks v4_12 as it wasn't exposed before that causing build breaks

Reply:
You mean manually?

Reply:
yeah